uniffi_bindgen/src/interface/universe.rs

```diff
@@ -111,7 +118,8 @@ impl TypeUniverse {
     /// Add many [`Type`]s...
     pub fn add_known_types(&mut self, types: TypeIterator<'_>) -> Result<()> {
         for t in types {
-            self.add_known_type(t)?
+            self.add_known_type(t)
+                .with_context(|| format!("adding type {t:?}"))?
```

Comment:
Are these getting overridden by the `parsing udl file...` context?  That what it seems like from the trybuild tests, but I'm not sure.